addons/l10n_ar_website_sale/static/tests/tours/portal_edit_information_tour.js

```diff
@@ -0,0 +1,24 @@
+/** @odoo-module **/
```

Comment:
No need i guess it can be removed. LGTM

Reply:
@kpt-odoo Changes are done,
Thank you :+1: